lib/get-client.js

```diff
@@ -95,19 +95,14 @@ const handler = (retry, limit, globalThrottler, endpoint) => ({
 
 module.exports = ({
   githubToken,
-  githubUrl,
-  githubApiPathPrefix,
+  githubUrl = '',
+  githubApiPathPrefix = '',
   retry = DEFAULT_RETRY,
   limit = RATE_LIMITS,
   globalLimit = GLOBAL_RATE_LIMIT,
 }) => {
-  const {port, protocol, hostname} = githubUrl ? url.parse(githubUrl) : {};
-  const github = new Octokit({
-    port,
-    protocol: (protocol || '').split(':')[0] || undefined,
-    host: hostname,
-    pathPrefix: githubApiPathPrefix,
-  });
+  const baseUrl = url.resolve(githubUrl, githubApiPathPrefix) || undefined;
```

Comment:
We had several issues with `url.resolve` used to concat url parts. I don't remember exactly but it was behaving differently depending on if the first part ends with a `/` or not.
On other plugin we use [url-join](https://github.com/jfromaniello/url-join) to concat URLs.

Also the code might do something unexpected if `githubUrl` is not set but  `githubApiPathPrefix` is.
I would suggest something like:
```js
const urljoin = require('url-join');

const baseUrl = githubUrl && urlJoin(githubUrl, githubApiPathPrefix);
```